modules/UI/side_pannels/SidePanelToggler.js

```diff
@@ -86,7 +86,7 @@ function toggle (object, selector, onOpenComplete,
         // before it's actually visible.
         // (Chrome seems to return {true} for an element which is in the DOM and
         // has non-null size set).
-        $(selector).css("width", "20%");
+        $(selector).css("width", "200px");
```

Comment:
Lucy, can you give some details on this change please? Why do we need to change the percentage value to a fixed pixels value?